src/sources/opentelemetry/http.rs

```diff
@@ -141,10 +127,12 @@ async fn handle_request(
                         Ok(protobuf(ExportLogsServiceResponse {}).into_response())
                     }
                     BatchStatus::Errored => Err(warp::reject::custom(Status {
+                        code: 2, // UNKNOWN - OTLP doesn't require use of status.code, but we can't encode a None here
```

Comment:
would it be beneficial at all to make a 

```Rust

const STATUS_CODE_UNKNOWN :  u32  = 2;  // or w/e the correct type is

```

?

Reply:
Probably slightly - I'm going to try and update the server to handle these better and we can return codes that may be actually helpful. So I'll hold off optimizing for a bit longer.